.travis.yml

```diff
@@ -1,59 +1,38 @@
-language: node_js
-node_js:
-  - "6"
+language: ruby
 
-before_install:
-  - "export CHROME_BIN=/usr/bin/google-chrome"
-  - "export DISPLAY=:99.0"
-  - "sh -e /etc/init.d/xvfb start"
-
-script:
-  - gulp build
-  - gulp test --coverage
-  - gulp docs
-  - gulp package
-  - gulp bower
-  - cat ./coverage/lcov.info | ./node_modules/.bin/coveralls
-
-notifications:
-  slack: chartjs:pcfCZR6ugg5TEcaLtmIfQYuA
-
-sudo: required
-dist: trusty
+os:
+  - linux
+  - osx
 
 addons:
-  chrome: stable
   firefox: latest
+  chrome: stable
+
+env:
+  - TRAVIS_NODE_VERSION=6
 
-# IMPORTANT: scripts require GITHUB_AUTH_TOKEN and GITHUB_AUTH_EMAIL environment variables
-# IMPORTANT: scripts has to be set executables in the Git repository (error 127)
-# https://github.com/travis-ci/travis-ci/issues/5538#issuecomment-225025939
+before_install:
+  - export CHROME_BIN=/usr/bin/google-chrome
+  - export DISPLAY=:99.0
+  - if [ "$TRAVIS_OS_NAME" != "osx" ]; then sh -e /etc/init.d/xvfb start; fi
+  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then ( sudo Xvfb :99 -ac -screen 0 1024x768x8; echo ok )& fi
 
-deploy:
-- provider: script
-  script: ./scripts/deploy.sh
-  skip_cleanup: true
-  on:
```

Comment:
You could leave the deploy block in place and just add something along the lines of:

    on:
        all_branches: true
        condition: `uname -s` == "Linux"